code/modules/cargo/bounties/chef.dm

```diff
@@ -26,25 +27,20 @@
 	reward = 3142
 	wanted_types = list(/obj/item/reagent_containers/food/snacks/pie)
 
+/datum/bounty/item/gardencook/khinkali
+	name = "Khinkali"
+	description = "Requesting -some khinki stuff- for a private staff party at Centcom."
+	reward = 2400
```

Comment:
Way to much like 1000~ is more fair for 2 meat slices.

Reply:
You still have to put in work to make it, low cost bounties are why people don't even bother doing them. Why put in effort when you can kill a single monkey and get more?

Reply:
There isn't a single thing you can really purchase with 1000 credits to begin with. If you fill out a bounty you should be able to purchase at least 1 useful crate with those credits. 2400 is enough for an insuls or engineer supply crate, for example.